dashboard/plugins/s3_usage/s3_stats.py

```diff
@@ -0,0 +1,193 @@
+import boto3
```

Comment:
My biggest concern is that this one file contains a lot of logic split into 4 classes. Those classes seem dependent on each other, so it seems that we may leave it this way, however, I think  that splitting it into separate files would have been beneficial. Your call @szczeles

Reply:
You're right, code got a bit bigger that I planned :D I will separate it to backed service (refresher task) and frontend service (query mechanisms)